routes/app.js

```diff
@@ -37,14 +37,17 @@ module.exports = function () {
 
     router.get('/admin', authHandler.isAuthenticated, function (req, res) {
         res.render('app/admin', {
-            admin: (req.user.role == 'admin')
+            admin: (req.user.role == 'admin'),
+            securityRating: req.query.securityRating
         })
     })
 
-    router.get('/admin/usersapi', authHandler.isAuthenticated, appHandler.listUsersAPI)
+    router.get('/admin/usersapi/', authHandler.isAuthenticated, authHandler.isAdmin, appHandler.listUsersAPI)
```

Comment:
In my first PR that changed to this file (before the refactor) , I passed the security rating in the endpoint like so: `/admin/usersapi/:securityRating`. This made it much easier to debug the two different security ratings in postman. The insecure endpoint is `/admin/usersapi/0` and the secure endpoint is  `/admin/usersapi/1`. I think I prefer that way (the older way before he refactor). What do you think?